lib/config/presets/index.ts

```diff
@@ -150,6 +150,10 @@ export async function getPreset(
     presetName,
     baseConfig,
   });
+  // istanbul ignore if
+  if (!presetConfig) {
+    throw new Error('dep not found');
```

Comment:
We should add a constant for this value as it's use on a lot places.

We should add a test too, even if it's trivial.

Reply:
Need to think about validation error message if preset resolving fails here. What would undefined `presetConfig` mean mean from user perspective? Seems like `getPreset()` interface is more like "return result or throw something".

Reply:
Okay, I've reused `PRESET_DEP_NOT_FOUND` constant. There are more strings to refactor, but let's do it out of this PR